observation_portal/common/configdb.py

```diff
@@ -553,7 +558,7 @@ def get_exposure_overhead(self, instrument_type, binning, readout_mode=''):
             # if the binning is not found, return the default binning (Added to support legacy 2x2 Sinistro obs)
             return default_mode['overhead'] + camera_type['fixed_overhead_per_exposure']
 
-        raise ConfigDBException(f'Instrument type {instrument_type} not found in configdb.')
+        raise InstrumentNotFoundException(f'Instrument type {instrument_type} not found in configdb.')
```

Comment:
I'm thinking for this method and the next it might be better to return a default and log a warning that the instrument type wasn't found so a default was used. Then we shouldn't even need the later changes to catch this exception